doc/userDocs/usersGuide.md

```diff
@@ -1551,6 +1550,23 @@ As a result, when using node labels it is recommended that you label all of your
 worker nodes with either `wvrole=driver` or `wvrole=sut`.  This will give you the 
 best control over pod placement.
 
+### Inter-pod affinity and anti-affinity<a name="pod-affinity"></a>
+
+With the Weathervane 2.1 release, configuration parameters were provided to manage the addition of pod affinity and anti-affinity rules with weights to the Weathervane pod manifests. 
+This provides more control for which nodes the pods are eligible to be scheduled on.
```

Comment:
"which nodes the pods are eligible to be scheduled on": Is the affinity setting a "hint/best effort" or a requirement for pod scheduling? If it's the latter, are there implications for the kubernetes cluster, as in number of nodes, size of nodes, etc and is that the reason for the change to the Application Resource Requirements table? Does the kubernetes resource requirement change if a non-default affinity setting is selected?

Reply:
The kubernetes docs use the terms "should" and "should not", which I take to mean best-effort.
Either way there are implications because packing the boxes (nodes) differently may result in the same contents (pods) no longer able to fit or able to fit and leaving different sized spaces that other stuff can now fit into. The summed value resource requirement should not change with different affinity, but if 'fixed-size' pods no longer fit due to 'fragmentation' of the node available resources, more resources may be required.  There may also be performance implications due to locality.  
I changed the application resources table because the values were not correct anymore, and didn't match the spreadsheet values (before or after I made minor updates).

Reply:
"This provides more control for which nodes the pods are eligible to be scheduled on" -> recommended "This changes how the Weathervane pods are scheduled across Kubernetes nodes"

In regards to the best effort vs requirement question, the Kubernetes documentation [here](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) states "you can indicate that the rule is "soft"/"preference" rather than a hard requirement, so if the scheduler can't satisfy it, the pod will still be scheduled;" so it sounds like this is something you can set. Are the affinity labels requiredDuringSchedulingIgnoredDuringExecution and preferredDuringSchedulingIgnoredDuringExecution relevant here?

Great explanation Ben. Based on this explanation, here are a few ideas of things that sound like they should be included:
"Changing the affinity setting will result in noncomparable performance results."
"Changing the affinity setting could result in different resource requirements at the Kubernetes node level, such as different CPU or memory resources requirements per node or requiring a different number of nodes to run the same number of Weathervane Application Instances."

Reply:
Updated. Does this look better?